core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -55,7 +57,7 @@ public LocalPageStore(LocalPageStoreOptions options) {
   public void put(PageId pageId, byte[] page) throws IOException {
     Path p = getFilePath(pageId);
     if (!Files.exists(p)) {
-      Files.createDirectories(p.getParent());
+      Files.createDirectories(Preconditions.checkNotNull(p.getParent(), "parent"));
```

Comment:
nit: for src code readability & more explicit error message
```
Preconditions.checkNotNull(p.getParent(), "Path for page %s can not be root", pageId);
Files.createDirectories(p.getParent());
```

Reply:
Done.